app/authentication/auth.py

```diff
@@ -5,8 +5,16 @@
 from sqlalchemy.exc import DataError
 from sqlalchemy.orm.exc import NoResultFound
 
+from app.dao.api_key_dao import get_api_key_by_secret
 from app.dao.services_dao import dao_fetch_service_by_id_with_api_keys
 
+JWT_AUTH_TYPE = 'jwt'
+API_KEY_V1_AUTH_TYPE = 'api_key_v1'
+AUTH_TYPES = [
+    ('Bearer', JWT_AUTH_TYPE),
+    ('ApiKey-v1', API_KEY_V1_AUTH_TYPE),
```

Comment:
Decide if this is a good auth type. People will need to put this in the `Authorization` header (`Authorization: ApiKey-v1 $api_key`)